zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/thrift/RemoteInterpreterService.java

```diff
@@ -6135,7 +6135,7 @@ public String toString() {
       }
       first = false;
       if (!first) sb.append(", ");
-      sb.append("st:");
+      sb.append("st:\n");
```

Comment:
Is it an intended change? because this file is an autogenerated file from RemoteInterpreterService.thrift